packages/editor/src/components/block-switcher/index.js

```diff
@@ -61,15 +61,11 @@ export class BlockSwitcher extends Component {
 				return null;
 			}
 			return (
-				<Toolbar>
-					<IconButton
-						disabled
-						className="editor-block-switcher__no-switcher-icon"
-						label={ __( 'Block icon' ) }
-					>
+				<div className="components-toolbar" aria-hidden="true">
+					<span className="components-icon-button editor-block-switcher__no-switcher-icon is-presentational">
```

Comment:
Same as in the toolbar case we should avoid the usage of components-icon-button class as it is internal to another component in another package.

Reply:
@jorgefilipecosta @tofumatt All set here as well. Let me know if you find anything else. Thanks for the feedback!